teps/README.md

```diff
@@ -0,0 +1,6 @@
+# Kubernetes Enhancement Proposals (TEPs)
+
+A Tekton Enhancement Proposal (TEP) is a way to propose, communicate and coordinate on new efforts for the Tekton project.
+You can read the full details of the project in [TEP-1](0001-tekton-enhancement-proposal-process.md).
+
+This process is still in a _beta_ state and is mandatory for all enhancements beginning release 1.0.
```

Comment:
what is release 1.0?

Reply:
it would be the first GA of tekton pipelines… But I need to rework this part, as it's not yet clear what we will call GA for tekton :upside_down_face: